docs/general/builtins/aggregation.rst

```diff
@@ -631,6 +631,39 @@ Example::
     executions of the aggregate function on the same data produce slightly
     differing results.
 
+.. _aggregation-stddev-samp:
+
+``stddev_samp(column)``
+-----------------------
+
+The ``stddev_samp`` aggregate function computes the `Sample Standard Deviation`_
+of the set of non-null values in a column. It is a measure of the variation
+of data values. A low standard deviation indicates that the values tend to be
+near the mean.
+
+``stddev_samp`` is defined on all numeric types and on timestamp. It always returns
```

Comment:
Does this really support large numeric values? Looks like it always uses a double representation (Number.doubleValue) while iterating. When trying this with e.g.:
```
        Object result = executeAggregation(new NumericType(null, null),
            new Object[][]{{new BigDecimal("1.7976931348623159E+308")}, {new BigDecimal("9223372036854775807")}});
```
a NULL is returned while PG returns the correct large number.

Reply:
Wouldn't it make sense to add numeric support within this PR? Like always adding numeric support for newly added aggs? But no strong opinion, it just feels a bit strange to add a new feature which includes a wrong documentation and/or is not complete...

Reply:
Makes sense yes, I will work on adding NUMERIC support on both stddev.

Reply:
Pushed support for NUMERIC.